rasa/nlu/registry.py

```diff
@@ -126,7 +130,7 @@ def pipeline_template(s: Text) -> Optional[List[Dict[Text, Text]]]:
     if components:
```

Comment:
I think it should be 
`def pipeline_template(s: Text) -> Optional[List[Dict[Text, Any]]]:`

Reply:
Fair enough, I didn't think about this. The parameter value can be anything from a string to a list of ints, so I'll change the type to `Any`.